hello_world.cpp

```diff
@@ -0,0 +1,7 @@
+// Note: this file is just to mark this project as C++ related.
+#include <iostream>
+
+int main()
+{
+    std::cout << "Hello, world!\n";
```

Comment:
I can read. but for the reason to be recognized as C++ on github, no need to have any objections to reflect or close.